pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -39,6 +40,9 @@ const (
 
 	// See https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2570-memory-qos
 	DefaultMemoryThrottlingFactor = 0.8
+
```

Comment:
not related to this PR, but curious if there should be a different default for Windows. cc: @marosset

Reply:
I'll take a look and can address any needed changes in a different PR.
Thanks @SergeyKanzhelev for asking about this.

Reply:
I created https://github.com/kubernetes/kubernetes/issues/105102 to track this.

Reply:
sorry, I meant `DefaultKubeletRunDir`

Reply:
> not related to this PR, but curious if there should be a different default for Windows. cc: @marosset

It would be better to omit an API-level default than to set OS-specific defaults. See the problems OS-specific API defaulting causes in https://github.com/kubernetes/kubernetes/pull/104376#issuecomment-899532470